interface/main/calendar/add_edit_event.php

```diff
@@ -116,9 +126,16 @@
     $collectthis = $collectthis[array_keys($collectthis)[0]]["rules"];
 }
 ?>
+<?php $disabled = (!$edit && ($_GET['group']==="true" || $_GET['group']==true ))?' disabled=true; ':'';?>
```

Comment:
empty only works null or empty string  and when  $_GET['group'] === "false"  this is not quite what i am looking for here because it will return true and it means that I am using groups .